test/test_downstream.py

```diff
@@ -286,7 +286,8 @@ def test_next_try_push_infra_fail_try_rebase(env, git_gecko, git_wpt, pull_reque
                     try_push.infra_fail = True
                     try_push["stability"] = False
 
-                assert sync.next_action == downstream.DownstreamAction.try_push_stability
+                assert sync.next_action == downstream.DownstreamAction.try_rebase
+                sync.next_try_push()
```

Comment:
I think you should add the mock data here `sync.next_try_push(try_cls=MockTryCls)`, then the test should pass.